README.md

```diff
@@ -106,21 +108,35 @@ If you you get a `could not connect to server` error, you could be experiencing
 psql -d foia -c "CREATE USER foia WITH PASSWORD '<<PASSWORD>>';"
 ```
 
+* Change database password in local_settings.py:
```

Comment:
It seemed like a more important part of setup than running the server. I can take it out.